graylog2-server/src/main/java/org/graylog/integrations/notifications/types/SlackEventNotification.java

```diff
@@ -130,6 +136,13 @@ SlackMessage createSlackMessage(EventNotificationContext ctx, SlackEventNotifica
                 String tag = config.notifyChannel() ? "channel" : "here";
                 template = StringUtils.f("@%s\n%s", tag, template);
             }
+            if (config.includeEventProcedure() && ctx.eventDefinition().isPresent()) {
```

Comment:
@ryan-carroll-graylog I noticed we have a license check in the frontend, do we want to add the same check here in the backend as well?

Reply:
Good call out @danotorrey , can't see it here but on the next line inside of `EventProcedureService.getDecoratedForEvent()` we do do a security license check, but lmk if you see any issues with that